### DISCUSSION TITLE
Dog blips out of existence.

### DISCUSSION THREAD

**Discussion by MoonlightNightwing** [DashieDev/DoggyTalentsNext#75] [Q&A] [answered]
Playing 1.18.2 in a modpack I had one pup that just blipped out of existence when I went out on an expedition peditoin and died. I went back to that place and the dog wasn't anywhere I searched the entire area spamming the heel command and walked back in the same direction I came spamming the whistle. Still nothing so I went into creative to use the creative mode tracker and was told that there is no such dog. Kinda sad now, can I get her back or not?

**Comment by DashieDev** [comment, marked as answer]
Did you try using /dog revive?

**Reply by MoonlightNightwing** [reply]
I just recently started using the mod so I didn't know about that command. It worked, thank you!
